tests/test_flow/test_copy.py

```diff
@@ -0,0 +1,138 @@
+import pytest
+import pickle
+from pathlib2 import Path
+from subprocess import check_call
+from ..helpers import (
+    skip_unless_gcs, GCS_TEST_BUCKET, df_from_csv_str, equal_frame_and_index_content, dask_py3_only
+)
+
+import bionic as bn
+import dask.dataframe as dd
+
+
+@pytest.fixture(scope='function')
+def preset_builder(builder):
+    builder.assign('x', 2)
+    builder.assign('y', 3)
+
+    @builder
+    def f(x, y):
+        return x + y
+
+    return builder
+
+
+@pytest.fixture(scope='function')
+def flow(preset_builder):
+    return preset_builder.build()
+
+
+@pytest.fixture(scope='function')
+def expected_dask_df():
+    df_value = df_from_csv_str('''
+    color,number
+    red,1
+    blue,2
+    green,3
+    ''')
+    return dd.from_pandas(df_value, npartitions=1)
+
+
+@pytest.fixture(scope='function')
+def dask_flow(builder, expected_dask_df):
+
+    @builder
+    @bn.protocol.dask
+    def dask_df():
+        return expected_dask_df
+
+    return builder.build()
+
+
+def test_copy_filename(flow):
+    assert pickle.loads(flow.get('f', mode='FileCopier')
```

Comment:
Sorry, I'm confused about what this is testing.  No copying is happening here, right?

Reply:
Yeah, this test is not copying and unnecessary since we have all the tests for local copying anyway.  Gonna delete!